frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/vm-wizard-storage-row.tsx

```diff
@@ -63,13 +64,13 @@ export type VMWizardNicRowProps = {
 };
 
 export const VmWizardStorageRow: React.FC<VMWizardNicRowProps> = ({
-  obj: { name, wizardStorageData, ...restData },
+  obj: { name, wizardStorageData, isCD, ...restData },
```

Comment:
I created `VMCDsTable` but regarding the row. why can't we make it generic?

Reply:
because it is basically a wrapper which just uses DiskSimpleRow or CDSimpleRow component. It adds nothing useful to it - the row has different columns.